### DISCUSSION TITLE
Is there a VIM colorscheme that looks like bat?

### DISCUSSION THREAD

**Discussion by JoaquimEsteves** [sharkdp/bat#2800] [Q&A] [answered]
First of all, allow me to thank you for providing us with such lovely free software.

One of my favorite things about bat is the way color is handled by default.

I like it so much I'd like to use it with my (N)VIM setup - but have so far failed to capture the correct colors.

Has someone gone to the trouble of setting up a vim compatible colorscheme?

**Comment by eth-p** [comment, marked as answer]
It might be a bit difficult to get the exact same highlighting<sup>1</sup>, but you can definitely find or modify a similar theme. The default theme used by `bat` is Monokai, and [this Monokai vim theme](https://github.com/ku1ik/vim-monokai) looks like it's using a similar color palette.

<sup>1</sup> Bat uses Sublime's syntax definitions, while Vim uses its own. The definitions will often label tokens differently, which makes `bat`/`vim` pick different completely colors from the theme palette.

**Reply by JoaquimEsteves** [reply]
Thank you kindly - precisely what I wanted.

For future reference [there's also a neovim version](https://github.com/polirritmico/monokai-nightasty.nvim) which includes some LSP shenanigans.
